custom_components/wyzeapi/cover.py

```diff
@@ -0,0 +1,188 @@
+            DeviceTypes.CAMERA
+        ]:
+            raise HomeAssistantError(f"Invalid device type: {self._camera.type}")
+        
+        self._camera_service = camera_service
+        self._available = self._camera.available
+        self._out_of_sync_count = 0
+        
+    @property
+    def device_info(self):
+        """Return device information about this entity."""
+        return {
+            "identifiers": {(DOMAIN, self._camera.mac)},
+            "name": f"{self._camera.nickname}.cgdc",
+            "connections": {
+                (
+                    dr.CONNECTION_NETWORK_MAC,
+                    self._camera.mac
+                )
+            }
+        }
+    @property
+    def extra_state_attributes(self):
+        """Return device attributes of the entity."""
+        return {
+            ATTR_ATTRIBUTION: ATTRIBUTION,
+            "device model": f"{self._camera.product_model}.{self._camera.device_params['dongle_product_model']}"
+        }
+    @property
+    def entity_registry_enabled_default(self) -> bool:
+        # always true because garage controllers are always connected to the cam
+        return True
+    
+    @property
+    def should_poll(self) -> bool:
+        return False
+    
+    @token_exception_handler
+    async def async_open_cover(self, **kwargs):
+        """Open the cover."""
+        try:
+            await self._camera_service.garage_door_open(self._camera)
+        except (AccessTokenError, ParameterError, UnknownApiError) as err:
+            raise HomeAssistantError(f"Wyze returned an error: {err.args}") from err
+        except Exception as err:
+            raise HomeAssistantError(err) from err
+        else:
+            self._camera.garage = True
+            self._out_of_sync_count = 0
+            self.async_write_ha_state()
+    
+    @token_exception_handler
+    async def async_close_cover(self, **kwargs):
+        """Close the cover."""
+        try:
+            await self._camera_service.garage_door_close(self._camera)
+        except (AccessTokenError, ParameterError, UnknownApiError) as err:
+            raise HomeAssistantError(f"Wyze returned an error: {err.args}") from err
+        except Exception as err:
+            raise HomeAssistantError(err) from err
+        else:
+            self._camera.garage = False
+            self._out_of_sync_count = 0
+            self.async_write_ha_state()
+
+    @property
+    def is_closed(self):
+        """Return if the cover is closed."""
+        return self._camera.garage == False
+        
+    @property
+    def available(self):
+        """Return the connection status of this cover."""
+        return self._camera.available
+    
+    @property
+    def supported_features(self):
+        """Flag supported features."""
+        return CoverEntityFeature.OPEN | CoverEntityFeature.CLOSE
+                
+    @property
+    def unique_id(self):
+        return f"{self._camera.mac}_GarageDoor"
+    @property
```

Comment:
Add a space here, and docstring.

```suggestion

    @property
```